js/popup.js

```diff
@@ -0,0 +1,136 @@
+  },
+  {
+    title: 'Multi-Post Stories',
+    devs: ['Facebook', ' • Full Stack Dev', ' • 2018'],
+    description:
+      'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. ',
+    tags: ['html', 'css', 'ruby', 'javascript'],
+    image: 'asset/Snapshoot-Portfolio.svg',
+  },
+
+  {
+    title: 'Facebook 360',
+    devs: ['Facebook', ' • Full Stack Dev', ' • 2020'],
+    description:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.",
+    tags: ['html', 'css', 'ruby', 'javascript'],
+    image: 'asset/Snapshoot-Portfolio1.svg',
+  },
+  {
+    title: 'Uber Navigation',
+    devs: ['Uber', ' • Lead Developer', ' • 2022'],
+    description:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.",
+    tags: ['html', 'css', 'ruby', 'javascript'],
+    image: 'asset/Snapshoot-Portfolio2.svg',
+  },
+];
+
+const workpage = document.querySelector('#workpage');
+for (let i = 0; i < projects.length; i += 1) {
+  workpage.innerHTML += `
+           <div class="work-box">
+                <div class="work-img">
+                    <img src="${projects[i].image}" alt="image one">
+                </div>
+                <div class="work-info">
+                    <h2 class="work-title">${projects[i].title}</h2>
+                    <div class="work-meta">
+                        <p class="work-para">canopy</p>
+                        <img src="asset/Counter.svg" alt="dot">
+                        <p class="work-para1">Back End Dev</p>
+                        <img src="asset/Counter.svg" alt="dot">
+                        <p class="work-para2">2015</p>
+                    </div>
+                    <div class="work-note">
+                        <p>${projects[i].description}</p>
+                    </div>
+                    <div class="work-tag">
+                        <ul>
+                        ${projects[i].tags
+    .map((j) => `<li> <div class="tag">${j}</div></li>`)
+    .join('')}
+                        </ul>
+                    </div>
+                    <button type="button" id="btn-${i}" class="see-project">See Project</button>
+                </div>
+            </div>
+           `;
+}
+
+const modal = document.getElementById('modal1');
+
+// const btn = document.querySelectorAll(".see-project");
+// btn.forEach((b) => {
+//   b.addEventListener("click", () => {
+//     modal.style.display = "block";
+//   });
+// });
```

Comment:
- [ ] [OPTIONAL] I kindly suggest that you remove the commented out lines of code here because they do not make your code neat and it isn't a [best practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md)

Reply:
thanks, I really appreciate the feedback I will rectify it